lib/tasks/pgbackups_archive.rake

```diff
@@ -1,8 +1,14 @@
+
 namespace :pgbackups do
 
   desc "Capture a Heroku PGBackups backup and archive it to Amazon S3."
-  task :archive do
-    PgbackupsArchive::Job.call
+
+  task :archive, [:app, :database] do |t, args|
+
+    args.with_defaults(app: ENV['PGBACKUPS_APP'])
```

Comment:
I think that both of the `args.with_defaults` lines should be removed so that the job class can be the single point of default assignment.  This will keep things DRY and prevent situations like the one going on right now (where the default set here for the database differs from the one set in the job class - which does what this does but then also falls back from there to `DATABASE_URL`).